nf_core/__main__.py

```diff
@@ -200,36 +201,40 @@ def launch(pipeline, id, revision, command_only, params_in, params_out, save_all
 
 
 # nf-core download
+
+
 @nf_core_cli.command(help_priority=3)
-@click.argument("pipeline", metavar="<pipeline name>")
+@click.argument("pipeline", required=False, metavar="<pipeline name>")
 @click.option("-r", "--release", type=str, help="Pipeline release")
 @click.option("-o", "--outdir", type=str, help="Output directory")
 @click.option(
     "-c",
     "--compress",
     type=click.Choice(["tar.gz", "tar.bz2", "zip", "none"]),
-    default="tar.gz",
     help="Archive compression type",
 )
 @click.option("-f", "--force", is_flag=True, default=False, help="Overwrite existing files")
-@click.option("-s", "--singularity", is_flag=True, default=False, help="Download singularity images")
 @click.option(
-    "-c",
-    "--singularity-cache",
-    is_flag=True,
-    default=False,
-    help="Don't copy images to the output directory, don't set 'singularity.cacheDir' in workflow",
+    "-i",
```

Comment:
This may just be me being slow, but `-i` for `--container`?

Reply:
Ah, competing with `--compress`. My suggestion would be to pick `-c` for `--container` and require long option for `--compress`.

Reply:
Yes maybe that's better, it grated on me too. Compress got there first so was trying not to break backwards comparability, but lots changing here so may as well make it as we want it.

`-x` for compress? That then competes with another one but the other one was pretty random.